app/routes/missions.agile.tsx

```diff
@@ -0,0 +1,5 @@
+import { redirect } from '@remix-run/react'
+
+export async function loader() {
+  return redirect('../archival/agile')
```

Comment:
```suggestion
  return redirect('../archival/agile', 301)
```

301 means [moved permanently](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301).